test/uninstall.test.ts

```
@@ -10,7 +9,7 @@ import { spawnSync } from "node:child_process";
 
 const UNINSTALL_SCRIPT = path.join(import.meta.dirname, "..", "uninstall.sh");
 
-function createFakeNpmEnv(tmp) {
+function createFakeNpmEnv(tmp: string): NodeJS.ProcessEnv {
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)